app/src/main/kotlin/com/wire/android/ui/authentication/devices/DeviceItem.kt

```diff
@@ -199,7 +200,24 @@ private fun DeviceItemTexts(
     }
 
     Spacer(modifier = Modifier.height(MaterialTheme.wireDimensions.removeDeviceItemTitleVerticalPadding))
-    val details: String = if (!device.registrationTime.isNullOrBlank()) {
+
+    device.mlsPublicKeys?.values?.firstOrNull()?.let { mlsThumbprint ->
+        Text(
+            style = MaterialTheme.wireTypography.subline01,
+            color = MaterialTheme.wireColorScheme.labelText,
+            maxLines = 1,
```

Comment:
Should it really be `maxLines = 1`?

I see it ends up cutting the thumbprint. Shouldn't we make sure that the whole thing is visible?

Reply:
this is `DeviceItem` from the list of all my devices (the first pair of screenshots), if you want to see the whole key - go to device screen :)